spec/classes/rcs_spec.rb

```diff
@@ -0,0 +1,81 @@
+require_relative '../spec_helper'
+
+describe 'rcs', :type => :class do
+
+  it { is_expected.to compile.with_all_deps }
+  it { is_expected.to contain_class('rcs::config') }
+
+  context 'default parameters' do
+    it do
+      is_expected.to contain_file('/etc/default/rcS') \
+        .with_content(/TMPTIME=0/)
+        .with_content(/SULOGIN=NO/)
+        .with_content(/DELAYLOGIN=NO/)
+        .with_content(/UTC=YES/)
+        .with_content(/VERBOSE=NO/)
+        .with_content(/FSCKFIX=NO/)
+    end
+  end
+
+  context 'tmptime is set to 7 days' do
+    let(:params) {{
+      :tmptime => '7',
+    }}
+    it do
+      is_expected.to contain_file('/etc/default/rcS') \
+        .with_content(/TMPTIME=7/)
+    end
+  end
+
+  context 'sulogin is set to yes' do
+    let(:params) {{
+      :sulogin => 'yes',
```

Comment:
They appear to work in Vagrant, but as of https://github.com/alphagov/puppet-rcs/commit/195e40c375850d742937d81135b16fd673bdfaaf, they'll always be upcased. That's tested in https://github.com/alphagov/puppet-rcs/commit/ff255235ea0a5090a25c21b7dfdebf29b808ea15.